### PR TITLE
[document] add java formatter in doc

### PR DESCRIPTION
**Pull request by jackwener**
## Proposed changes

Now there isn't the guidance of java format. We should add it in doc.

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)
- [ ] Optimization. Including functional usability improvements and performance improvements.
- [ ] Dependency. Such as changes related to third-party components.
- [ ] Other.

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have created an issue on (Fix #ISSUE) and described the bug/feature there in detail
- [ ] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If these changes need document changes, I have updated the document
- [ ] Any dependent changes have been merged

## Further comments

None

### CODE REVIEW

**Review by EmmyMiao87 — DISMISSED** [review]
LGTM

File: docs/zh-CN/developer-guide/java-format-code.md [hunk 1]
```diff
@@ -45,3 +45,14 @@ javax
...
 * 禁止使用 `import *`
 * 禁止使用 `import static`
+
+## Checkstyle
+
+现在的 `CI` 之中会有 `formatter-check` 进行代码格式化检测。
+
+如果使用 `IDEA` 进行 Java 开发，请在设置中安装 `Checkstyle-IDEA` 插件。
+
+在 `Tools->Checkstyle` 的 `Configuration File` 里点击 `Use a local Checkstyle file`，选择项目的 `checkstyle.xml` 文件。
```
Comment (morningman) [inline docs/zh-CN/developer-guide/java-format-code.md:55]:
```suggestion
在 `Tools->Checkstyle` 的 `Configuration File` 里点击 `Use a local Checkstyle file`，选择项目的 `fe/checkstyle.xml` 文件。
```

**Comment by jackwener** [comment]
Fixed

**Comment by jackwener** [comment]
Thanks for your review

**Review by morningman — APPROVED** [review]

**Pull request merged**